docs/api.swagger.yaml

```diff
@@ -1376,8 +1376,8 @@ paths:
   /warden/groups:
     get:
       description: >-
-        The subject making the request needs to be assigned to a policy
-        containing:
+        The subject making the request, if member is specified, needs to be
```

Comment:
As discussed in my comment, I'd like to revert this and replace this with a policy check against `rn:hydra:warden:groups`